### ISSUE TITLE
question about training trcik

### ISSUE THREAD

**Issue by yyknight** [yulunzhang/RCAN#89]
Thanks for your impressive work. But I have several doubts about your results. I have retrained your network RCAN several times, but I cannot remake the results 33.34dB on Urban100/BSD100 dataset  for x2 (Set5, Set14 can achieve), I can just achieve around 33.29dB (highest PSNR with more than 1000 epochs). Are there some training tricks or something slightly influence the results?

**Comment by 1187697147** [comment]
HI,@yyknight,  have you trained the RCAN using the default command?
Which validation set do you use?
